src/System.Data.SqlClient/src/Configurations.props

```diff
@@ -0,0 +1,24 @@
+﻿<Project>
```

Comment:
Remove all Configurations.props files from this PR. That's old infra from corefx 2.1 that we don't support anymore.